tests/test_version.py

```diff
@@ -0,0 +1,29 @@
+#
+# Tests the basics of chaste_codegen
```

Comment:
No it doesn't :D

Reply:
uups I sometimes don't know which file I'm looking at :D